x/claim/keeper/claim_test.go

```diff
@@ -344,3 +344,62 @@ func (suite *KeeperTestSuite) TestClaimOfDecayed() {
 		test.fn()
 	}
 }
+
+func (suite *KeeperTestSuite) TestClawbackAirdrop() {
+	suite.SetupTest()
+
+	tests := []struct {
+		name           string
+		address        string
+		sequence       uint64
+		expectClawback bool
+	}{
+		{
+			name:           "airdrop address active",
+			address:        "osmo1000644etcp4k4awz77kvy3pryplv8ky8py962d",
+			sequence:       1,
+			expectClawback: false,
+		},
+		{
+			name:           "airdrop address inactive",
+			address:        "osmo10008uvk6fj3ja05u092ya5sx6fn355wayx20rq",
+			sequence:       0,
+			expectClawback: false,
+		},
+		{
+			name:           "non airdrop address inactive",
```

Comment:
```suggestion
			name:           "non airdrop address active",
```

This needs an actual test for an inactive account. @mattverse can you add that here?